python/app.py

```diff
@@ -0,0 +1,60 @@
+from flask import Flask, render_template, request, abort
+import os
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.core.credentials import AzureKeyCredential
+from azure.core.exceptions import ClientAuthenticationError, ServiceRequestError
+
+
+app = Flask(__name__)
+
+def create_text_analytics_client():
+    # Set a default key if missing to avoid TypeError on use
+    key = os.getenv('CS_ACCOUNT_KEY', '')
+    endpoint = f"https://{os.getenv('CS_ACCOUNT_NAME')}.cognitiveservices.azure.com/"
+    return TextAnalyticsClient(endpoint=endpoint, credential=AzureKeyCredential(key))
+
+def language_detection(text):
+    if not text:
+        return {}
+
+    try:        
+        client = create_text_analytics_client()
+        response = client.detect_language(documents=[text])
+        if response and response[0].primary_language:
+            return response[0]
+    except ClientAuthenticationError as e:
+        # CS_ACCOUNT_NAME environmnet variable missing or CS_ACCOUNT_KEY is invalid
```

Comment:
```suggestion
        # CS_ACCOUNT_NAME environment variable missing or CS_ACCOUNT_KEY is invalid
```

Reply:
Eh, I don't both try/except for this situation, since it shouldnt happen in production. This should just be a 500, as the problem is with the app, not with the user. (400 implies user error). So the user should see a standard server error, and the app admin should check logs for the issue.

I know we like to have frontends give more info for our samples, but thats not the best production practice, so my compromise is that I show the Python error class in the frontend, with a message that they should check logs for more details and stack trace.